### PR TITLE
profiles: use.desc: Add USE=keyring, replaces gnome-keyring,libsecret,secret et al

### PR DESCRIPTION
**Pull request by a17r**

### CODE REVIEW

**Comment by voyageur** [comment]
Pending on happy CI but thanks for tackling this one! Looks good for wmudmount from quick check

File: app-text/evince/evince-43.1-r1.ebuild [hunk 1]
```diff
@@ -101,7 +101,7 @@ src_configure() {
 		-Duser_doc=true
 		$(meson_use introspection)
 		-Ddbus=true
-		$(meson_feature gnome-keyring keyring)
+		$(meson_feature keyring keyring)
```
Comment (thesamesam) [inline app-text/evince/evince-43.1-r1.ebuild:104]: Simplify

File: profiles/arch/hppa/use.mask [hunk 2]
```diff
@@ -154,6 +154,7 @@ gconf
 gnome
 gnome-keyring
```
Comment (thesamesam) [inline profiles/arch/hppa/use.mask:155]: Do some consumers remain?

**Review by thesamesam — APPROVED** [review]

File: profiles/arch/hppa/use.mask (hunk 2 above)
Comment (a17r) [reply profiles/arch/hppa/use.mask:155]: 2 remain: `mate-base/mate-session-manager`, `sys-auth/pambase`, directly pulling in `gnome-base/gnome-keyring`.

File: app-admin/system-config-printer/files/system-config-printer-1.5.18-fix-debugprint-exception.patch [hunk 3]
```diff
@@ -0,0 +1,22 @@
+From 399b3334d6519639cfe7f1c0457e2475b8ee5230 Mon Sep 17 00:00:00 2001
```
[the remark this answers is not in the document]
Comment (thesamesam) [reply app-admin/system-config-printer/files/system-config-printer-1.5.18-fix-debugprint-exception.patch:1]: I don't agree (could easily be a hash of a patch someone else wrote) and it makes it more expensive to check the upstream status but fine.

File: profiles/arch/hppa/use.mask (hunk 2 above)
Comment (thesamesam) [reply profiles/arch/hppa/use.mask:155]: Thanks!

File: app-admin/system-config-printer/files/system-config-printer-1.5.18-fix-debugprint-exception.patch (hunk 3 above)
[the remark this answers is not in the document]
Comment (a17r) [reply app-admin/system-config-printer/files/system-config-printer-1.5.18-fix-debugprint-exception.patch:1]: But we don't add non-upstream patches, and if we do, that warrants extra information.

For the purpose of review of a future version bump, a short source comment will be added in the ebuild.

[the remark this answers is not in the document]
Comment (thesamesam) [reply app-admin/system-config-printer/files/system-config-printer-1.5.18-fix-debugprint-exception.patch:1]: We'll have to agree to disagree. It's still useful to have a link you can click to e.g. easily see which tag it's in.

File: profiles/arch/m68k/use.mask [hunk 4]
```diff
@@ -105,6 +105,7 @@ audit
 # Masking these globally, since we don't have X
 gnome
 gnome-keyring
```
[the remark this answers is not in the document]
Comment (a17r) [reply profiles/arch/m68k/use.mask:103]: Still necessary unless `gnome-keyring` flag vanishes completely.

**Pull request closed without merging**